types/index.d.ts

```diff
@@ -518,29 +560,43 @@ declare namespace math {
      * @param format The Matrix storage format
      * @returns The created Matrix
      */
-    matrix(format?: 'sparse' | 'dense'): Matrix;
+    matrix(format?: 'sparse' | 'dense'): Matrix
     /**
      * @param data A multi dimensional array
      * @param format The Matrix storage format
      * @param dataType The Matrix data type
      * @returns The created Matrix
      */
-    matrix(data: MathCollection, format?: 'sparse' | 'dense', dataType?: string): Matrix;
+    matrix(
+      data: MathCollection,
+      format?: 'sparse' | 'dense',
+      dataType?: string
+    ): Matrix
 
     /**
      * Create a number or convert a string, boolean, or unit to a number.
      * When value is a matrix, all elements will be converted to number.
      * @param value Value to be converted
      * @returns The created number
      */
-    number(value?: string | number | BigNumber | Fraction | boolean | MathCollection | Unit | null): number | MathCollection;
+    number(
+      value?:
+        | string
+        | number
+        | BigNumber
+        | Fraction
+        | boolean
+        | MathCollection
+        | Unit
+        | null
+    ): number | MathCollection
     /**
```

Comment:
It might be good to use some of the type definitions if possible to avoid such hugely long type unions that get spread out by Prettier to a whole bunch of super-short lines.

Reply:
I think this PR should generally limited to autofix + manually changes necessary to fix linting errors. But I agree that this would be a good change.

Reply:
Good idea indeed, but let's keep that for a separate PR